web/controllers/task_controller.ex

```diff
@@ -12,8 +12,8 @@ defmodule CoursePlanner.TaskController do
 
   def index(%{assigns: %{current_user: %{id: id, role: "Volunteer"}}} = conn, _params) do
     render(conn, "index_volunteer.html",
-      available_tasks: Tasks.get_unassigned_tasks(),
-      your_tasks: Tasks.get_user_tasks(id))
+      available_tasks: Tasks.get_unassigned(Timex.now()),
```

Comment:
what do you think about extracting the call to `Timex.now()` into a variable? the way it is now there might be a slightly difference between each call